src/pages/index.html

```diff
@@ -126,6 +132,137 @@
     <div class="cover-shadow__div">
     </div>
     <main class="sun__main">
+        <section class="hero__section">
+            <video src="../../assets/video/mv-pc.mp4" autoplay="" muted="" webkit-playsinline="" playsinline=""
+                loop=""></video>
+        </section>
+        <section class="layout__section"></section>
+        <section class="home__section">
+            <div class="wrapper__main">
+                <h2 class="home-title__heading">Our Business</h2>
+            </div>
+        </section>
+        <section class="home__section home__section--mb">
+            <div class="wrapper__main">
+                <div class="sun-main__content">
+                    <p class="bio__paragraph">
+                        As a Digital Creative Studio, Sun* always prioritize the spirit
+                        of owning the products, thinking creatively in any project to bring the most
+                        "<strong>awesome</strong>" experience to end-users. With two service lines namely
+                        "<strong>Creative &amp; Engineering</strong>" and "<strong>Talent Platform</strong>", Sun* has
+                        always been step by step with technology creating great value for society.
+                    </p>
+                    <div class="bio__image">
+                        <img src="https://sun-asterisk.vn/wp-content/uploads/2020/11/intro-business.png"
+                            alt="Sun* Inc. Recruitment">
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="home__section home__section--mb2">
+            <div class="wrapper__main intro__main">
+                <div class="business__wrapper">
+                    <li class="business__item business__item--active">
+                        <div class="business__sub-title">Digital Creative Studio</div>
+                        <div class="business__title">Creative <br>&amp; Engineering</div>
+                        <div class="business__description">Abundant human resource specializing in technology, design
+                            and
+                            business.</div>
+                        <a class="business__btn button--bg-white button--size-large button--round-large"
+                            href="https://sun-asterisk.vn/en/creative-engineering-english/" target="_blank"
```

Comment:
```suggestion
                            href="/#" target="_blank"
```
Tạm thời chưa có link e để là /#